sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -210,14 +211,16 @@ public <T> Mono<T> invokeMethod(InvokeMethodRequest invokeMethodRequest, TypeRef
       pathSegments.addAll(Arrays.asList(methodSegments));
 
       final Map<String, String> headers = new HashMap<>();
-      if (contentType != null && !contentType.isEmpty()) {
-        headers.put("content-type", contentType);
-      }
       headers.putAll(httpExtension.getHeaders());
       if (metadata != null) {
         headers.putAll(metadata);
       }
       byte[] serializedRequestBody = objectSerializer.serialize(request);
+      if (contentType != null && !contentType.isEmpty()) {
```

Comment:
I am fixing this due to the deprecation of content-type as json by default in runtime.